deploy/build.sh

```diff
@@ -17,6 +17,6 @@
 set -e
 set -x
 
-godep go build -a github.com/google/cadvisor
+make
```

Comment:
I'd prefer `make build` here. I don't think `fmt` `vet` and `test` need to be part of this script.